AutomationHelpers/src/UserCodeCollections/WebLibrary.cs

```diff
@@ -89,5 +91,50 @@ public static string GetHttpStatusCode(string url)
 
             return ((int)resp.StatusCode).ToString();
         }
+
+        /// <summary>
+        /// Takes screenshot of entire web page and reports it.
+        /// </summary>
+        /// <param name="repoItemInfo">Repository item</param>
+        [UserCodeMethod]
+        public static void ReportFullPageScreenshot(RepoItemInfo repoItemInfo)
+        {
+            try
+            {
+                if (repoItemInfo == null)
+                {
+                    Report.Error("Repository item must be provided.");
```

Comment:
Rather throw an ArgumentNullException, this should actually never happen unless used from code

Reply:
Done